omero/sysadmins/system-migration.rst

```diff
@@ -62,10 +62,12 @@ Restore the binary
 
 - Remove all the directories under ``/OMERO`` e.g. ``sudo rm -rf /OMERO/*``.
 
-- Change ownership and group of the directories under ``OMERO_BINARY_BACKUP`` (but do not change the ownership of the backup diretory containing the ``omero.config`` i.e. ``/tmp/OMERO_BINARY_BACKUP/backup``)::
+- Move the backup diretory containing the ``omero.config``, ``/tmp/OMERO_BINARY_BACKUP/backup``, out of ``OMERO_BINARY_BACKUP``
 
-    sudo chown -R omero-server /tmp/OMERO_BINARY_BACKUP/ManagedRepository/
-    sudo chgrp -R managed_repo_group /tmp/OMERO_BINARY_BACKUP/ManagedRepository/
+- Change ownership and group of the directories under ``OMERO_BINARY_BACKUP``::
+
+    sudo chown -R omero-server /tmp/OMERO_BINARY_BACKUP/
+    sudo chgrp -R managed_repo_group /tmp/OMERO_BINARY_BACKUP/
```

Comment:
Sorry, I just realized - not on all servers the ``managed_repo_group`` exist. This is specific for our training servers. I think we need to say something like ``managed_repo_group`` or another as appropriate - this means one needs to compare with what was there before - the groups on other servers can vary. On the demo server the group was even ``root``.

Reply:
Maybe the group ultimately does not matter, and we could then drop the ``chgrp`` line ?
Either you know this ^^^ or I would have to test it.